### PR TITLE
Updated cluster blueprint resource to facilitate create and delete of cluster blueprints

### PR DESCRIPTION
**Pull request by kjrach**

### CODE REVIEW

File: examples/resources/hpegl_caas_cluster_blueprint/resource.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  name = "BLR"
+  space_id = ""
+}
+
+resource hpegl_caas_cluster_blueprint testbp {
+  name         = "tf-clusterbp"
+  k8s_version  = ""
+  default_storage_class = ""
+  site_id = data.hpegl_caas_site.blr.id
+  cluster_provider = ""
```
Comment (parekhha) [inline examples/resources/hpegl_caas_cluster_blueprint/resource.tf:32]: Do we have to provider id here ?

File: examples/resources/hpegl_caas_cluster_blueprint/resource.tf [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  control_plane = {
+    name = "master"
+    machine_blueprint_id = ""
+    count = ""
+  }
+  worker_nodes {
```
Comment (parekhha) [inline examples/resources/hpegl_caas_cluster_blueprint/resource.tf:37]: THis is not correct it should be
worker_nodes = [ 
  {
  }
]

File: examples/resources/hpegl_caas_cluster_blueprint/resource.tf [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+resource hpegl_caas_cluster_blueprint testbp {
+  name         = "tf-clusterbp"
+  k8s_version  = ""
+  default_storage_class = ""
+  site_id = data.hpegl_caas_site.blr.id
+  cluster_provider = ""
+  control_plane = {
+    name = "master"
```
Comment (parekhha) [inline examples/resources/hpegl_caas_cluster_blueprint/resource.tf]: this should not require name.

File: internal/resources/schemas/machine_sets.go [hunk 4]
```diff
@@ -11,12 +11,12 @@ func MachineSets() map[string]*schema.Schema {
...
 			Type:     schema.TypeString,
 			ForceNew: true,
-			Computed: true,
+			Required: true,
 		},
 		"machine_blueprint_id": {
 			Type:     schema.TypeString,
 			ForceNew: true,
-			Computed: true,
+			Required: true,
```
Comment (parekhha) [inline internal/resources/schemas/machine_sets.go]: Is this schema used for cluster ? Do not mix schema across resources.

**Review by parekhha — CHANGES_REQUESTED** [review]

File: examples/resources/hpegl_caas_cluster_blueprint/resource.tf (hunk 1 above)
Comment (kjrach) [reply examples/resources/hpegl_caas_cluster_blueprint/resource.tf:32]: cluster_provider = "ecp"

File: examples/resources/hpegl_caas_cluster_blueprint/resource.tf (hunk 3 above)
Comment (kjrach) [reply examples/resources/hpegl_caas_cluster_blueprint/resource.tf]: Done

**Review by parekhha — APPROVED** [review]

**Pull request merged**